vispy/app/qt.py

```diff
@@ -14,13 +16,28 @@
                        "in use.")
 
 
-class QtCanvas(QtGui.QWidget):
-    """ Qt widget containing a vispy Canvas. 
-    
+if qt_lib == 'pyqt4':
+    from PyQt4 import QtGui
+    QWidget, QGridLayout = QtGui.QWidget, QtGui.QGridLayout  # Compat
+elif qt_lib == 'pyqt5':
+    from PyQt5 import QtGui, QtWidgets
+    QWidget, QGridLayout = QtWidgets.QWidget, QtWidgets.QGridLayout  # Compat
+elif qt_lib == 'pyside':
+    from PySide import QtGui
+    QWidget, QGridLayout = QtGui.QWidget, QtGui.QGridLayout  # Compat
```

Comment:
this can be combined with `pyqt4` as `if qt_lib in ('pyqt4', 'pyside')` right? should be more DRY